desktop/apps/article/templates/meta.jade

```diff
@@ -80,5 +80,11 @@ if !article.indexable
   meta( name='robots', content='noindex' )
 
 //- Slick styles
-<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" />
-<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css"/>
\ No newline at end of file
+link( rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css")
+link( rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css")
+
+//- Set Viewport scale
+if sd.IS_MOBILE
+  meta( name="viewport", content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" )
```

Comment:
Do we need maximum-scale here, or could initial scale work?
http://a11yproject.com/posts/never-use-maximum-scale/

Reply:
Hmm -- this is interesting. We've always prevented pinch-zoom on mobile (i believe for design reasons, see [here](https://github.com/artsy/force/blob/master/mobile/components/layout/templates/head.jade#L3)), but for accessibility reasons we _do_ want to enable it. I'll remove it for now and ask someone.